.github/workflows/production.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           echo "${{ steps.get-previous-release.outputs.previous_tag }}"
   prepare_for_release:
     needs: [tests, check_branch, get_version_number]
-    runs-on: macos-14
+    runs-on: macos-15
```

Comment:
![image](https://github.com/user-attachments/assets/d76211cb-6900-49b6-a8a1-ca6e8cc31efe)
Seems like it's in public preview, no stability concerns from you Jake?

Reply:
According to the road map macOS-latest should be macos 15 already https://github.com/github/roadmap/issues/986. I personally think it should be ok. If we run into issues I can revert it back